content/en/docs/concepts/security/index.md

```diff
@@ -118,13 +116,13 @@ work together with `istiod` to automate key and certificate
 rotation at scale. The following diagram shows the identity
 provisioning flow.
 
-{{< image width="75%"
+{{< image width="40%"
     link="./id-prov.svg"
-    caption="Identity Provision"
+    caption="Identity Provisioning Workflow"
     >}}
 
-Istio provisions identities through the
-[secret discovery service (SDS)](https://www.envoyproxy.io/docs/envoy/latest/configuration/security/secret#secret-discovery-service-sds)
+Istio Envoy provisions keys and certificates through the
```

Comment:
Maybe @frankbu has a thought, but I'm not sure `Istio Envoy` makes sense here. I'm thinking the original `Istio` is better.

Reply:
I agree with @ericvn 
```suggestion
Istio provisions keys and certificates through the
```